testsuite/tests/kuadrant/authorino/operator/tracing/test_tracing_tags.py

```diff
@@ -18,7 +18,6 @@ def authorino_parameters(authorino_parameters):
     return authorino_parameters
 
 
-@pytest.mark.issue("https://github.com/Kuadrant/authorino-operator/issues/171")
```

Comment:
Why?

Reply:
the issue is fixed and closed

Reply:
It is for regression purposes, every issue we link to will be closed